daprdocs/content/en/reference/cli/dapr-mtls/dapr-mtls-renew-certificate.md

```diff
@@ -0,0 +1,73 @@
+---
+type: docs
+title: "mtls renew certificate CLI command reference"
+linkTitle: "mtls renew certificate"
+description: "Detailed information on the mtls renew certificate CLI command"
+weight: 3000
+---
+
+### Description
+This command can be used to renew expiring Dapr certificates in Kubernetes cluster.
+It renews root CA certificate, issuer certificate and issuer key.
+
+### Supported platforms
+
+- [Kubernetes]({{< ref kubernetes >}})
+
+### Usage
+
+```bash
+dapr mtls renew-certificate [flags]
+```
+
+### Flags
+
+| Name           | Environment Variable | Default           | Description                                 |
+| -------------- | -------------------- | ----------------- | ------------------------------------------- |
+| `--help`, `-h` |                      |                   | help for renew-certificate
+| `--kubernetes`, `-k` |                      | `false` | supprted platform|                             |
```

Comment:
Ah! Perfect. You added the -k command for this! Excellent. Now we just need to add these for the expiry and export commands